### DISCUSSION TITLE
how to open free_snap_tap_3 without pressing enter to run script?

### DISCUSSION THREAD

**Discussion by stvnkng** [wasiejen/Free-Snap-Tap#49] [Q&A]
which text file should i edit or line ?

**Comment by wasiejen** [comment]
just add \<arg>-nomenu at the beginning of the fst config file or use the -menu start argument when you start it. the first option is the easier one :-D

```bash
<arg>-nomenu

# Tap Groups
a, d
w, s
# Rebinds
# Macros
```
